components/lists/list.html

```diff
@@ -0,0 +1,145 @@
+              <img
+                class="person-image"
+                src="https://via.placeholder.com/150"
+                alt="https://via.placeholder.com/150"
+              />
+            </div>
+            <div class="person-details">
+              <label class="person-name">Random Name</label>
+              <label class="person-mail">random@mail.com</label>
+            </div>
+          </div>
+        </li>
+        <li>
+          <div class="list-card">
+            <div class="person-img">
+              <img
+                class="person-image"
+                src="https://via.placeholder.com/150"
+                alt="https://via.placeholder.com/150"
+              />
+            </div>
+            <div class="person-details">
+              <label class="person-name">Random Name</label>
+              <label class="person-mail">random@mail.com</label>
+            </div>
+          </div>
+        </li>
+        <li>
+          <div class="list-card">
+            <div class="person-img">
+              <img
+                class="person-image"
+                src="https://via.placeholder.com/150"
+                alt="https://via.placeholder.com/150"
+              />
+            </div>
+            <div class="person-details">
+              <label class="person-name">Random Name</label>
+              <label class="person-mail">random@mail.com</label>
+            </div>
+          </div>
+        </li>
+        <li>
+          <div class="list-card">
+            <div class="person-img">
+              <img
+                class="person-image"
+                src="https://via.placeholder.com/150"
+                alt="https://via.placeholder.com/150"
+              />
+            </div>
+            <div class="person-details">
+              <label class="person-name">Random Name</label>
+              <label class="person-mail">random@mail.com</label>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+    <br /><br /><br />
+    <!-- normal-list-->
+    <div class="normal-list normal-list-wrap">
+      <ul>
+        <li>item&nbsp;1</li>
+        <li>item&nbsp;2</li>
+        <li>item&nbsp;3</li>
+        <li>item&nbsp;4</li>
+        <li>item&nbsp;5</li>
+        <li>item&nbsp;6</li>
+      </ul>
+      <ul class="square-bullet">
+        <li>item&nbsp;1</li>
+        <li>item&nbsp;2</li>
+        <li>item&nbsp;3</li>
+        <li>item&nbsp;4</li>
+        <li>item&nbsp;5</li>
+        <li>item&nbsp;6</li>
+      </ul>
+      <ul class="lower-alpha-bullet">
+        <li>item&nbsp;1</li>
+        <li>item&nbsp;2</li>
+        <li>item&nbsp;3</li>
+        <li>item&nbsp;4</li>
+        <li>item&nbsp;5</li>
+        <li>item&nbsp;6</li>
+      </ul>
+      <ul class="upper-roman-bullet">
+        <li>item&nbsp;1</li>
+        <li>item&nbsp;2</li>
+        <li>item&nbsp;3</li>
+        <li>item&nbsp;4</li>
+        <li>item&nbsp;5</li>
+        <li>item&nbsp;6</li>
+      </ul>
```

Comment:
incorporate margins for better readability

Reply:
Working on that. Thanks for informing

Reply:
please request changes and not comment them